colossalai/inference/executor/rpc_worker.py

```diff
@@ -179,19 +181,18 @@ def _init_model(self, model_or_path: Union[nn.Module, str], model_policy: Policy
         """
 
         if isinstance(model_or_path, str):
-            # is_local = os.path.isdir(model_or_path)
+            is_local = os.path.isdir(model_or_path)
             try:
-                hf_config = AutoConfig.from_pretrained(model_or_path, trust_remote_code=True)
+                hf_config = AutoConfig.from_pretrained(model_or_path, trust_remote_code=True, torch_dtype=self.dtype)
                 arch = getattr(hf_config, "architectures")[0]
```

Comment:
What if the input model name or path is not supported yet? Please refer to the usage in `engine.py`

Reply:
We will not call the remote worker if the engine doesn't support this model.
The logic you mentioned is in `rpc_engine` instead of `rpc_worker`.
If in rpc_worker, the error message will show many times(there are many rpc_workers. @yuanheng-zhao